src/mongocxx/test/spec/util.cpp

```diff
@@ -222,11 +222,17 @@ void set_up_collection(const client& client,
     auto db = client[test[database_name].get_string().value];
     db.drop();
 
+    bsoncxx::builder::basic::document opts;
+    bsoncxx::builder::basic::document opts;
```

Comment:
It looks like a goofed merge caused this variable declaration to be duplicated.

Reply:
Haha, nice! Fixed 😄